lib/blocks.php

```diff
@@ -77,6 +77,8 @@ function gutenberg_reregister_core_block_types() {
 				'post-comments-count.php'       => 'core/post-comments-count',
 				'post-comments-form.php'        => 'core/post-comments-form',
 				'post-comments-link.php'        => 'core/post-comments-link',
+				'comments-query.php'            => 'core/comments-query',
+				'comments-template.php'         => 'core/comments-template',
```

Comment:
We discussed the naming with @ntsekouras and we arrived at the following to match with the Query Loop block:
```suggestion
				'comments-query-loop.php'    => 'core/comments-query-loop',
				'comment-template.php'          => 'core/comment-template',
```

Sidenote: in the code there is `core/query` that has the title Query Loop. It was too late to rename it to `core/query-loop` that would work best.

Reply:
By the way, there is no PHP file for the Comment Template block. We should add one with a basic handling for the comments query loop.